src/api/app/controllers/webui/groups/bs_requests_controller.rb

```diff
@@ -28,6 +42,27 @@ def set_group
       def request_method
         REQUEST_METHODS[params[:dataTableId]]
       end
+
+      def set_bs_request
+        @bs_requests = @user_or_group.requests
```

Comment:
I would add the `return unless Flipper.enabled?(:request_index, User.session)` check in here as done in the other `bs_requests_controller`. Saves a query incase of not using the beta version

Reply:
Fair enough